scripts/ci/automation_full_test.py

```diff
@@ -33,7 +33,7 @@
 azdev_test_result_dir = os.path.expanduser("~/.azdev/env_config/mnt/vss/_work/1/s/env")
 python_version = os.environ.get('PYTHON_VERSION')
 job_name = os.environ.get('JOB_NAME')
```

Comment:
```suggestion
python_version = os.environ.get('PYTHON_VERSION', 'default_py')
job_name = os.environ.get('JOB_NAME', 'default_job')
```
Will it be better to set default value for used parameters?